### PR TITLE
Add ZirkoniaMC.net

### PR DESCRIPTION
**Pull request by NiceKype**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by NiceKype** [comment]
background
background@2x
icon
icon@2x
logo
logo@2x

File: minecraft_servers/zirkoniamc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "server_name": "zirkoniamc",
+  "nice_name": "ZirkoniaMC",
+  "direct_ip": "play.zirkoniamc.net",
+  "server_wildcards": [
+    "%.zirkoniamc.net"
```
Comment (DoJapHD) [inline minecraft_servers/zirkoniamc/manifest.json:6]: Not a real wildcard
https://i.imgur.com/j8aqJh1.png

File: minecraft_servers/zirkoniamc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://zirkoniamc.net",
+    "web_shop": "https://shop.zirkoniamc.net",
+    "web_support": "https://zirkoniamc.net/support",
```
Comment (DoJapHD) [inline minecraft_servers/zirkoniamc/manifest.json:15]: Not available

File: minecraft_servers/zirkoniamc/manifest.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      "name": "OneBlock",
+      "command": "/oneblock",
+      "url": "https://www.zirkoniamc.net/wiki/oneblock/",
+      "color": "#9A00EE",
+      "versions": "1.20<*"
+    }
+  },
+  "discord": {
+    "server_id": 985929292162691072,
+    "rename_to_minecraft_name": true
```
Comment (DoJapHD) [inline minecraft_servers/zirkoniamc/manifest.json:46]: Reserved for partners, change to false

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server is not yet released; reopen PR once server is available

**Pull request closed without merging**